<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stock</class>
 <widget class="QDialog" name="stock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_title_Form">
     <property name="maximumSize">
      <size>
       <width>291</width>
       <height>84</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="ImgReport_2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Recource.qrc">:/rec/img/Form.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbReport_2">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Форма Складов</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QTableWidget" name="twOrg">
     <property name="minimumSize">
      <size>
       <width>521</width>
       <height>531</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>521</width>
       <height>531</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>441</width>
       <height>311</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>441</width>
       <height>311</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="leID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbName_2">
        <property name="text">
         <string>Адрес склада</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leAdress"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbID_2">
        <property name="text">
         <string>Номер слада </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_3">
     <property name="maximumSize">
      <size>
       <width>491</width>
       <height>44</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAdd_2">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit_2">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDel_2">
        <property name="text">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove_2">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>351</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>41</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад </string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTextEdit" name="teResult">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>83</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Recource.qrc"/>
 </resources>
 <connections/>
</ui>
